submissions/chabuChabu/html/product.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8" >
+        <title>Products-page</title>
+    </head>
+    <body >
+
+
+        <h1>Kwathu</h1><hr>
+
+         <div >
+         	<h3>SAMSUNG  2020 73"Q85T QLED 4K HDR</h3>
+         </div>
+
+
+                
+         <table >
+            
+             <tr >
+                
+                
+                <td>
+                    <img src="product-01.jpg"><br>
+      
+                </td>
+                
+                
+                 <td>
+                     <img src="product-02.jpg"><br>
+                  
+                </td>
+                 
+
+             </tr>
+         </table>
+
+         <h3><em>Price</em> :<strong>ZMW 8888</strong><br></>
+
+         <h3><strong>Prouct description</strong></h3>
+
+         <table>
+            
+             <tr>
+                
+                 <td>
+                 	<h3>Screen Technology</h3>
+                	<ul>
+                       <li> LED TV Screen.</li>
+                       <li> 4K Ultra HD Certified display resolution</li>
+                       <li> HDR10+ - makes whites brighter, intensifies colour and enhances the overall picture for an</li>
+                       <li> Resolution 3840 x 2160 pixels</li>
+                         
+                   </ul>
+      
+                </td>
+                <td>
+                	<h3>Dimesions</h3>
+                	<ul>
+                       <li> Screen size: 75 inches.</li>
+                       <li> Size of TV H95.8, W167.3, D6cm.</li>
+                       <li> Size of TV with stand: H104.8, W167.3, D34.1cm.</li>
+                       <li> Width of TV stand 142.7cm.</li>
+                       <li> Weight of TV without stand 30.4kg (unpackaged)</li>   
+                   </ul>
+                 
+                </td>
+              
+             </tr>
+          </table> 
+          <h2><strong>Checkout similar products</strong></h2>
+
+          <table>
+            
+             <tr>
+                <td>
+                  
+                   <ol>
+
+                     <li><a href="#">Samsung 47" SmartTV</a></li>
+                     <li><a href="#">Samsung 57" SmartTV</a></li>
+                     <li><a href="#">Samsung 60" SmartTV</a></li>
+                   </ol>
+             
+                 </td> 
+
+              </tr>
+         </table>
+        <!--<footer>
```

Comment:
Remove this unused comments here @ChabuChabu

Reply:
Noted